generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-dialog.component.html.ejs

```diff
@@ -22,7 +22,7 @@ _%>
 <form name="editForm" role="form" novalidate (ngSubmit)="save()" #editForm="ngForm">
 
     <div>
-        <h2  id="my<%= entityClass %>Label" jhiTranslate="<%= keyPrefix %>home.createOrEditLabel">Create or edit a <%= entityClassHumanized %></h2>
+        <h4  id="my<%= entityClass %>Label" jhiTranslate="<%= keyPrefix %>home.createOrEditLabel">Create or edit a <%= entityClassHumanized %></h4>
```

Comment:
Why this change?

Reply:
Failed: No element found using locator: By(css selector, .jh-create-entity) in travis

Reply:
after remove modal html is like this 
```
<div class="modal-header">
        <h4 class="modal-title" id="myOperationLabel" jhiTranslate="jhipsterSampleApplicationApp.operation.home.createOrEditLabel">Create or edit a Operation</h4>
        <button type="button" class="close" data-dismiss="modal" aria-hidden="true"
                (click)="clear()">&times;</button>
   </div> 
```

and modal will remove

Reply:
all travis build that failed i thin for css and html 
https://travis-ci.org/jhipster/generator-jhipster/jobs/360669286#L5005

Reply:
Well, maybe it's the test that should be changed. 

To me, it does not seem right from a structural point of view, to have a h4 nested under a h1.

In React,  it's also h2 https://github.com/jhipster/generator-jhipster/blob/master/generators/entity-client/templates/react/src/main/webapp/app/entities/entity-detail.tsx.ejs#L65

Reply:
where is this test ? what will be change ? can you help me to change that ?

Reply:
See https://github.com/azizkhani/generator-jhipster/blob/master/generators/entity-client/templates/angular/src/test/javascript/e2e/entities/entity.spec.ts.ejs#L85
These are the templates to generate e2e tests

Reply:
Travis generates about 20 projects then builds them, run unit tests and end to end tests.

Reply:
so either you run these Travis" builds locally, all of them but it takes time or you run only the ones that have failed in real Travis. or you re-generate your sample and run yarn e2e

Reply:
Are you on Windows?